docs/product-installation-guide/src/main/asciidoc/controller-eap-run-proc.adoc

```diff
@@ -0,0 +1,28 @@
+[id='controller-eap-run-proc']
+= Running the Controller
+After you have installed the Controller on {EAP}, use this procedure to run the Controller.
+
+.Prerequisite
+The Controller installed and configured in an `__EAP_HOME__`
+
+.Procedure
+. In a terminal application, navigate to `__EAP_HOME__/bin`.
+. Enter the following command:
+** On Linux or UNIX-based systems:
++
+[source,bash]
+----
+$ ./standalone.sh
+----
+** On Windows:
++
+[source,bash]
+----
+standalone.bat
+----
+
+. In a web browser, enter the value of the `org.kie.server.controller` property, for example `http://localhost:8080/kie-wb/rest/controller`.
+[Note]
+====
+Alternatively, you can use the {KIE_SERVER} Java API Client to access the Controller.
```

Comment:
Should it mean that users can use Java client to communicate with the Controller?